wallet/res/layout/wallet_transactions_fragment.xml

```diff
@@ -34,15 +34,14 @@
             android:orientation="horizontal"
             android:paddingStart="16dp"
             android:paddingLeft="16dp"
-            android:paddingTop="8dp"
             android:paddingEnd="16dp"
-            android:paddingRight="16dp"
-            android:paddingBottom="8dp">
+            android:paddingRight="8dp">
 
             <ImageView
                 android:id="@+id/transaction_filter_btn"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
+                android:padding="8dp"
```

Comment:
This button is a bit small and hard to hit. Added padding and reduced margin to make the clickable area bigger while keeping the looks.